openpgp/packet/signature.go

```diff
@@ -420,6 +422,12 @@ func parseSignatureSubpacket(sig *Signature, subpacket []byte, isHashed bool) (r
 		}
 	case prefKeyServerSubpacket:
 		sig.PreferredKeyServer = string(subpacket[:])
+	case issuerFingerprint:
+		raw := subpacket[1:]
+		l := len(raw)
+		fp := make([]byte, l)
+		copy(fp, raw)
```

Comment:
If the goal is to just copy the slice to some owned memory, you can use `append([]byte{}, raw...)` to save a line here.